### ISSUE TITLE
Marketplace Coming Soon Page

### ISSUE THREAD

**Issue by jackson-laskey** [unbounded-enterprise/sample-app#38]
Rather than taking the user to the dashboard, it would nice if we could direct them to a page similar to screenshot below when a user clicks on the marketplace link in the nav bar.

![Image](https://user-images.githubusercontent.com/25757801/231529366-6bbde611-ca6a-4319-bfdf-1790df1b2f04.png)

**Comment by jackson-laskey** [comment]
@dstraffon Any thoughts on using a more generic image rather than Duro Dogs here? I'm open either way... I think most projects would just remove the marketplace page at this point anyways since the functionality isn't there yet

**Comment by dstraffon** [comment]
good catch. think you're right that'll probably be removed by most, but still think it's smart to replace the DD logo with something more generic just to avoid confusion for users who might assume it is a "duro dog sample app" rather than asset layer based.

attaching an alternative option below


![Image](https://user-images.githubusercontent.com/25757801/232575790-5c9d4dbe-51f4-4633-a3c5-31eab925d4ee.png)


![Image](https://user-images.githubusercontent.com/25757801/232575803-4fd723dd-7970-4fb1-bb4b-22354ab31577.png)
